### DISCUSSION TITLE
支持OA信息对接

### DISCUSSION THREAD

**Discussion by Baili-BL** [shepherdpp/qteasy#151] [Show and tell]
大佬后面会支持交易买卖点信息对接OA工具吗？

**Comment by shepherdpp** [comment]
不好意思，没有明白您的意思。能说的具体一点吗？

**Reply by Baili-BL** [reply]
比如一些执行的交易信号，发送到飞书/钉钉/微信。（类似发送到webhook）;跟supermind/聚宽平台差不多。将信号发送到应用里面。

**Reply by shepherdpp** [reply]
这个可以有！而且不难实现
会加入到开发RoadMap中去，不过可能需要等一段时间。
我现在的工作重点还是在优化内核（这应该是最后一次大规模优化系统内核了），以确保内核更高效、而且彻底实现策略数据与运行时机的完全自由定义。如果能找到共同开发者，进度应该能快一些
